gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemTest.java

```diff
@@ -54,6 +56,20 @@ public void before() throws IOException {
     postCreateInit();
   }
 
+  @Test
+  public void verifyHadoopPath() throws Exception {
+    GoogleHadoopFileSystem eagerFs =
+        GoogleHadoopFileSystemTestHelper.createInMemoryGoogleHadoopFileSystem();
+    String fileNameWithColon = "empty:file";
+    FSDataOutputStream stream =
```

Comment:
nit: use try-with-resource to close stream automatically

Reply:
Wouldn't even need it closing it immediately as it's an empty file.